cmd/skaffold/app/cmd/config/util.go

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright 2018 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubernetes/context"
+
+	homedir "github.com/mitchellh/go-homedir"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	yaml "gopkg.in/yaml.v2"
+)
+
+const defaultConfigLocation = ".skaffold/config"
```

Comment:
will this work on windows?

Reply:
it should, I see windows-specific logic in `github.com/mitchellh/go-homedir` which is what I'm using here along with `filepath.Join`

Reply:
Do you need to do a filepath.Join() on this? Should it be `.skaffold\config` on windows?

Reply:
good catch, I should have thought about that when I responded :)